pkg/unvanquished_src.dpkdir/ui/options_graphics.rml

```diff
@@ -226,6 +226,11 @@
 				</row>
 
 				<h2><translate>Particles</translate></h2>
+				<row>
+					<h3><translate>Marks</translate></h3>
+					<input cvar="cg_marks" type="checkbox" />
+					<p><translate>Allow marks to be painted on surfaces.</translate></p>
```

Comment:
How about examples to make this clearer. `Draw marks (such as bullet holes or creep) on surfaces`

Reply:
Done.